docs/operation/volume_workflow/index.rst

```diff
@@ -0,0 +1,16 @@
+Volume Management Workflow
+==========================
+This section highlights **MetalK8s Volume Management Workflow** which entails
+volume creation, volume deletion and volume provisioning neccessary for use in
```

Comment:
`volume creation and volume deletion`, I don't think we cover ` volume provisioning neccessary for use in persistent data storage within a MetalK8s Cluster.`